libraries/adaptive-expressions/src/index_browser.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @module adaptive-expressions
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+export * from './expressionFunctions';
```

Comment:
Consider having index_common.ts that exports things shared between browser and node.  Have both index.ts and index_browser.ts export * from index_common.  That way folks don't have to try and keep the common stuff in sync between the two files